src/main/java/com/questhelper/helpers/achievementdiaries/karamja/KaramjaHard.java

```diff
@@ -416,7 +424,10 @@ public List<PanelDetails> getPanels()
 		shortcutSteps.setLockingStep(usedShortcutTask);
 		allSteps.add(shortcutSteps);
 
-		PanelDetails eatOomlieWrapSteps = new PanelDetails("Eat Oomlie Wrap", Collections.singletonList(eatOomlie),
+		PanelDetails eatOomlieWrapSteps = questHelperPlugin.getPlayerStateManager().getAccountType().isAnyIronman()
+			? new PanelDetails("Eat Oomlie Wrap", Collections.singletonList(eatOomlie),
+			cooking50, oomlieWrap)
+			: new PanelDetails("Eat Oomlie Wrap", Collections.singletonList(eatOomlie),
```

Comment:
I think it would be preferred to not make any changes to this, but instead add a section-specific 50 cooking requirement for the oomlie wrap section.

Since you don't need 50 cooking for eating it, only for making it (which we don't help out with)